test/e2e/multicast_test.go

```diff
@@ -141,7 +152,118 @@ func TestMulticast(t *testing.T) {
 		if !multipleInterfacesFound {
 			t.Skip("Skipping test because none of the Nodes has more than one multicast enabled interface")
 		}
-		runTestMulticastForwardToMultipleInterfaces(t, data, nodeIdx, 3464, "224.3.4.13", nodeMulticastInterfaces[nodeIdx])
+		testMulticastForwardToMultipleInterfaces(t, data, nodeIdx, 3464, "224.3.4.13", nodeMulticastInterfaces[nodeIdx])
+	})
+	t.Run("testMulticaststats", func(t *testing.T) {
+		skipIfNumNodesLessThan(t, 2)
+		testcases := []multicastStatsTestcase{
+			{
+				name: "testMulticastNetworkPolicyStats",
+				senderConfigs: []senderConfigs{
+					{nodeName: nodeName(0), name: "test2-sender-1", IPs: []string{"225.20.2.2", "225.20.2.3"}, sendSessions: 10},
+				},
+				multicastANPConfigs: []ANPConfigs{
+					{
+						name:         "anp1-multicast",
+						appliedToPod: "test2-sender-1",
+						ruleConfigs: []ruleConfig{
+							{name: "allowtwothreemulticast", address: "225.20.2.3", action: crdv1alpha1.RuleActionAllow},
+							{name: "droponetwomulticast", address: "225.20.2.2", action: crdv1alpha1.RuleActionDrop},
```

Comment:
hard to get what the names mean, what does one stand for?
If the name is for ANP rule, it supports upper case and "-" and perhaps ".", make it more clear?

Reply:
updated.